src/components/controls/MediaStreamControl.tsx

```diff
@@ -24,14 +49,30 @@ export class MediaStreamControl extends React.PureComponent<IMediaControlProps,
         )
     }
 
+    private getAudioEnabled(): boolean {
+        return this.props.stream.getAudioTracks()[0].enabled;
+    }
+
+    private setAudioEnabled(enabled: boolean) {
+        return this.props.stream.getAudioTracks()[0].enabled = enabled;
+    }
+
+    private getVideoEnabled(): boolean {
+        return this.props.stream.getVideoTracks()[0].enabled;
```

Comment:
add a check to make sure `getVideoTracks[0] !== undefined`

Reply:
Done